aiorazemax/drivers.py

```diff
@@ -0,0 +1,82 @@
+import asyncio
+import json
+import logging
+from typing import Optional
+
+import aiobotocore
+
+from aiorazemax.exceptions import DriverError
+
+
+class Message:
+    def __init__(self, id, event_name, receipt_handle, body):
+        self.id = id
+        self.event_name = event_name
+        self.body = body
+        self.receipt_handle = receipt_handle
+
+
+class SQSDriver:
+    def __init__(self, sqs_client, sqs_queue_url):
+        self._client = sqs_client
+        self._queue_url = sqs_queue_url
+
+    async def close(self):
+        await self._client.close()
+
+    async def receive_message(self) -> Optional[Message]:
+        messages = await self._client.receive_message(
+            QueueUrl=self._queue_url,
+            MaxNumberOfMessages=1,
+            WaitTimeSeconds=1
+        )
+
+        logging.debug(f"Message received: {messages}")
+
+        if not messages or not messages.get('Messages'):
+            return None
+
+        message = messages['Messages'][0]
+        message = self._process_message(message)
+
+        return message
+
+    async def mark_message_processed(self, message: Message) -> None:
+        await self._client.delete_message(
+            QueueUrl=self._queue_url,
+            ReceiptHandle=message.receipt_handle
+        )
+        logging.debug("Message {} deleted".format(message.id))
+
+    def mark_message_unprocessed(self, message_id: Message, exception: Exception):
+        # TODO: Add dead letter queue implementation
+        pass
+
+    @classmethod
+    def _process_message(cls, message_sqs) -> Message:
```

Comment:
Is this message_sqs dict documented somewhere? maybe we could add a link to the documentation to avoid searching for it every time
Maybe this could be in a different PR